G/gismo/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+using BinaryBuilder
+using Pkg
+
+name = "gismo"
+version = v"24.08.0"
+sources = [
+    GitSource("https://github.com/gismo/gismo.git",       # The URL of the git repository
+              "fac4a95e60825bbc34014a524759e1178c9be646") # The commit hash to checkout
+]
+
+# NOTE: to control nproc, use the environment variable BINARYBUILDER_NPROC=<number of processors>
+script = raw"""
+cmake -B build                                      \
+  `# cmake specific`                                \
+  -DCMAKE_INSTALL_PREFIX=${prefix}                  \
+  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}  \
+  -DCMAKE_BUILD_TYPE=Release                        \
+  -DGISMO_OPTIONAL="gsCInterface"                   \
+  -DGISMO_gsCInterface_HEAD=""                      \
+  -DGISMO_WITH_OPENMP=ON                            \
+  -DTARGET_ARCHITECTURE=none                        \
+  -DCMAKE_CXX_STANDARD=11                           \
```

Comment:
You shouldn't need to specify the C++ standard: if they package fails to do so then it's in their build system that should be reported and fixed upstream, not something we should deal with.